include/mvdtool/mvd_base.hpp

```diff
@@ -91,16 +94,17 @@ class File : public MVDFile {
 };
 
 
-
-
-
-inline MVDType::MVDType _mvd_format(const std::string & filename) {
+inline MVDType::MVDType _mvd_format(const std::string& filename) {
     using boost::algorithm::ends_with;
     // mvd2
-    const std::string mvd_ext = ".mvd2";
     if (ends_with(filename, ".mvd2")) {
         return MVDType::MVD2;
     }
+    // tsv
```

Comment:
I don't know what @matz-e thinks, but to me this wrongly leads another developer that TSV is a storage format of MVD, as if TSV was standalone...
Printing a TSV doesn't have to be something this tool does (people can open it, even with excel)
It would be very useful, however, to dump combined info (mvd3+tsv, or sonata+tsv)

Reply:
I see.. I can get rid of the `tsv` printer then. Which should be the combined info though? All the info the `tsv` file includes? I see in the `print_csv` that only the following values are printed: `GID; POSITION_X; POSITION_Y; POSITION_Z; ROTATION_Q0; ROTATION_Q1; ROTATION_Q2; ROTATION_Q3;  MORPHO; MTYPE; ETYPE; SYNCLASS;`

Reply:
Do we need any printing functionality? We can always use the Python interface to explore files.

Reply:
Oh, Im sorry I thought there was already something in place that could be "extended".
Then I agree, let's not lose time. If you can reuse work and have summary() printing a summary of infos from the TSV, great, otherwise don't bother

Reply:
@ferdonline It's not something that will take a lot of time write `print_summary()`, it's only what information we want to include. But anyway, I think @matz-e is right. Since there is the python interface for getting all the `tsv` values, I think it's sufficient.